FLEDGE.md

```diff
@@ -1148,6 +1149,112 @@ Currency checking after `scoreAd()` happens only inside component auctions.  If
+
+**Views** don’t require any user interaction and can be registered via `<img>` or `<script>` tags using the `attributionsrc` attribute:
+
+```
+<img src="https://advertiser.example/pixel"
+     attributionsrc="'https://adtech.example/register-view?metadata=...">
+
+<script src="https://advertiser.example/register-view"
+        attributionsrc="'https://adtech.example/register-view?metadata=...">
+```
+
+**Clicks** can be registered on HTML anchor tags:
+
+```
+<a href="https://advertiser.example/landing"
+   attributionsrc="https://adtech.example/register-click?metadata=...">
+  click me
+</a>
+```
+
+**Clicks** can also be registered via calls to window.open that occur with [transient activation](https://html.spec.whatwg.org/multipage/interaction.html#transient-activation):
+
+```
+// Encode the attributionsrc URL in case it contains special characters,
+// such as '=', that will cause the parameter to be improperly parsed.
+window.open(
+    "https://advertiser.example/landing", "_blank",
+    attributionsrc=encodeURIComponent(
+        "https://adtech.example/register-click?metadata=..."));
+```
+
+When `attributionsrc` is used:
+ * The browser will always look for view and click data on the response to the request that goes to a clickthrough URL ("href" on an `<a>` tag) or impression URL ("src" on an `<img>` or `<script>` tag).
+ * When `attributionsrc` is non-empty, the browser will also send a background request to the URL specified in the value of `attributionsrc`, and then look for view and click data on the response.
+ * The `attributionsrc` HTML attribute may include multiple space-delimited URLs, in which case the browser will issue a background request for each.
+
+
+When the browser sends these request(s), it will include a new `Ad-Auction-Registration-Eligible` header whose value will be exactly one of "view" or "click".
+
+
+1. `<a>` and `window.open` will have the click value.
+1. Other APIs that automatically set `Ad-Auction-Registration-Eligible` (i.e. `<img>` and `<script>`) will have the view value.
+1. Requests from JavaScript using `window.fetch` may have the view value when issued using the fetch option, `{ eventSourceEligible: true }`.
+1. All other requests will not have the `Ad-Auction-Registration-Eligible` header.
+
+
+A server receiving the `Ad-Auction-Registration-Eligible` header may respond in such a way that indicates to the browser that this request should be counted among the events used to calculate that user's view and click counts. To do so, the server's response should include an `Ad-Auction-Register-Event` structured response header of the form:
+
+```
+Ad-Auction-Register-Event: type=[view|click]; eligible-origins=https://www.example-dsp.com,https://www.example2.com;
```

Comment:
Is the top-level type a [Dictionary](https://www.rfc-editor.org/rfc/rfc8941.html#name-dictionaries)?